src/pages/benchmarks/[benchmark_id]/index.tsx

```diff
@@ -137,6 +150,28 @@ const BenchmarkPage = () => {
   // BUG?: Sometimes if the user reloads/navigates away and confirms, the update has time to go through and data is saved. Is this something we should fix?
   useConfirmBeforeLeave(hasInputChanged);
 
+  const handleFileUpload = async (fileId: string) => {
+    if (!currentTrial) {
+      throw new Error("Trial data has not yet loaded");
+    }
+
+    await attachFileToTrialDataMutation.mutateAsync({
+      file_id: fileId,
+      trial_data_id: currentTrial?.trial_data_id,
+    });
+  };
+
+  const handleFileDelete = async (fileId: string) => {
+    if (!currentTrial) {
+      throw new Error("Trial data has not yet loaded");
+    }
+
+    await removeFileFromTrialDataMutation.mutateAsync({
```

Comment:
Correct me if I'm wrong, but looks like we're removing the association but not actually deleting the uploaded file, which would be concerning to me as a user. If so, we should update this to delete the file as well.

Reply:
88a3f23